packages/ui/src/components/molecules/NotesCard/NotesCard.tsx

```diff
@@ -27,16 +30,48 @@ interface NotesCardProps {
 }
 
 const NotesCard = ({title, data, style}: NotesCardProps) => {
+  const Colors = useThemeColor();
+
+  const [expanded, setExpanded] = useState(false);
+  const [chevronHeight, setChevronHeight] = useState(0);
+
   if (data == null || data === '') {
     return null;
   }
 
+  const toggleExpanded = () => {
+    setExpanded(current => !current);
+  };
+
   return (
     <View style={[styles.description, style]}>
       <Text style={styles.title}>{title}</Text>
-      <Card style={styles.note}>
-        <HtmlInput defaultInput={data} readonly={true} />
-      </Card>
+      <View
+        onLayout={event => {
+          const {height} = event.nativeEvent.layout;
+          setChevronHeight(height);
+        }}>
+        <Card
+          style={[
+            styles.note,
+            expanded ? styles.expandedNote : styles.collapsedNote,
```

Comment:
This condition and these styles are not required, you can only use the style condition in HtmlInput